Automattic/wp-calypso
# i18n: Use getters for contextual help items properties with translate/localizeUrl calls

#### Changes proposed in this Pull Request

* Remove re-building the entire context result objects on i18n `change` event.
* Use getter syntax for properties with with `translate` or `localizeUrl`, to ensure getting the computed value with the current i18n state whenever the property is being accessed.

#### Testing instructions

* Checkout locally and go to `http://calypso.localhost:3000/home`.
* Confirm links in the `Get help` section are being translated.
* Test with translation chunks disabled (`yarn start`) and translation chunks enabled (`BUILD_TRANSLATION_CHUNKS=true ENABLE_FEATURES=use-translation-chunks yarn start`).
